src/main/java/frc/robot/subsystems/ramp/states/ExtendingState.java

```diff
@@ -0,0 +1,22 @@
+package frc.robot.subsystems.ramp.states;
+
+import frc.robot.constants.JsonConstants;
+import frc.robot.subsystems.ramp.states.RampState.RampTriggers;
+
+public class ExtendingState extends RampState {
+
+  @Override
+  public void periodic() {
+    if (mechanism.inputs.position < JsonConstants.rampConstants.extendPosition) {
+      setVoltage(JsonConstants.rampConstants.extendingVoltage);
+    } else {
+      fireTrigger.accept(RampTriggers.HOME);
+    }
+    super.periodic();
+  }
+
+  @Override
+  public boolean inPosition() {
```

Comment:
I think if we are in this state we are never in position to do anything, so we should just return false